src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -207,12 +229,26 @@ export class RelayFeeCalculator {
 
     if (toBN(amountToRelay).eq(0)) return MAX_BIG_INT;
 
-    const getGasCosts = this.queries.getGasCosts(deposit, amountToRelay, relayerAddress).catch((error) => {
-      this.logger.error({ at: "sdk-v2/gasFeePercent", message: "Error while fetching gas costs", error });
-      throw error;
-    });
+    const getGasCosts = this.queries
+      .getGasCosts(deposit, simulateZeroFill ? bnOne : amountToRelay, relayerAddress)
+      .catch((error) => {
+        this.logger.error({
+          at: "sdk-v2/gasFeePercent",
+          message: "Error while fetching gas costs",
+          error,
+          simulateZeroFill,
+          deposit: JSON.stringify(deposit, jsonReplacerWithBigNumbers),
+        });
+        throw error;
+      });
     const getTokenPrice = this.queries.getTokenPrice(tokenInformation.symbol).catch((error) => {
-      this.logger.error({ at: "sdk-v2/gasFeePercent", message: "Error while fetching token price", error });
+      this.logger.error({
+        at: "sdk-v2/gasFeePercent",
+        message: "Error while fetching token price",
+        error,
+        simulateZeroFill,
+        deposit: JSON.stringify(deposit, jsonReplacerWithBigNumbers),
```

Comment:
`simulateZeroFill` and the complete `deposit` object aren't really necessary here - how about just logging the `destinationToken` ?

Reply:
Done